src/resource/resource.ts

```diff
@@ -176,6 +178,87 @@ export function isPodOwner(
   return podOwner === webId;
 }
 
+/**
+ * Get the URLs of Resources linked to the given Resource.
+ *
+ * Solid servers can link Resources to each other. For example, in servers
+ * implementing Web Access Control, Resources can have an Access Control List
+ * Resource linked to it via the `acl` relation.
+ *
+ * @param resource A Resource fetched from a Solid Pod.
+ * @returns The URLs of Resources linked to the given Resource, indexed by the key that links them.
+ * @since Not released yet.
+ */
+export function getLinkedResourceUrlAll(
+  resource: WithServerResourceInfo
+): internal_LinkedResourceUrls {
```

Comment:
If the type is part of the external API, is it quite right that it is prefixed `internal_` ? Is it until we agree on it and make it part of the official API ?

Reply:
No good catch, that would have resulted in annoying `any`s breaking type checking for developers. Fixed in 3f19b833.